src/slack-model.jl

```diff
@@ -1,6 +1,29 @@
 export SlackModel, SlackNLSModel
 
-get_slack_ind(jl, ind) = map(b -> findfirst(x -> x == b, ind), intersect(jl, ind))
+function get_slack_ind(jl, ind)
+  if intersect(jl, ind) == []
+    return Int[]
+  end
+  return map(b -> findfirst(x -> x == b, ind), intersect(jl, ind))
+end
+
+"""
+    get_relative_indices(model)
+
+Return two sets of triplet with the relative indices of `jlow`, `jupp` and `jrng`
```

Comment:
```suggestion
Return two sets of triplets with the relative indices of `jlow`, `jupp` and `jrng`
```

Reply:
NB: this function doesn't return two sets of triplets. A set of triplets would be `[(a, b, c), (d, e, f), ...]`.

Reply:
Thanks for the suggestions. I removed `two sets of triplets with`